ETH-Merge-Bots/Milestones-in-TTD/src/agent.spec.ts

```diff
@@ -0,0 +1,257 @@
+import { BlockEvent, HandleBlock } from "forta-agent";
+import { TestBlockEvent } from "forta-agent-tools/lib/test";
+import { provideHandleBlock } from "./agent";
+import BigNumber from "bignumber.js";
+import { createFinding, createFinalFinding } from "./findings";
+import { getNumberOfDays, getAverageBlockTime, getEstimatedNumberOfBlocksUntilMerge } from "./utils";

```

Comment:
Similar to the `createFinding`s functions, perhaps would be better to create local instances of the `utils` functions rather than using the imported ones?

If the function in the `utils` is wrong, the test could still pass as that is what is being used in the bot.

Reply:
done 👍